components/org.wso2.identity.event.http.publisher/src/main/java/org/wso2/identity/event/http/publisher/internal/component/ClientManager.java

```diff
@@ -105,11 +117,49 @@ public ClientManager() throws HTTPAdapterException {
                     config.getSocketTimeout() + ", maxConnections=" +
                     asyncConnectionManager.getMaxTotal() + ", maxConnectionsPerRoute=" +
                     asyncConnectionManager.getDefaultMaxPerRoute());
+
+            int poolSize = 10; // number of worker threads
```

Comment:
Shall we make these thread and pool properties configurable ?

Reply:
Lets check other occurrences as well ?

Reply:
Will configure them in the next pr as it will complicate the pr